src/components/Tabs/Tabs.test.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import {fireEvent, render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import Tabs from './index';
+
+const TestTabs = [
+  {
+    component: <span>Tab 1</span>,
+    label: 'FirstTab',
+  },
+  {
+    component: <span>Tab 2</span>,
+    label: 'SecondTab',
+  },
+];
+
+describe('Tabs component', () => {
+  test('renders with default testId Tabs', () => {
+    render(<Tabs tabs={TestTabs} latestReleaseNumber={1} />);
+    const tabs = screen.getByTestId('Tabs');
+    expect(tabs).toBeTruthy();
+  });
+
+  test('renders with className Tabs', () => {
+    render(<Tabs tabs={TestTabs} latestReleaseNumber={1} />);
+    const tabs = screen.getByTestId('Tabs');
+    expect(tabs).toHaveClass('Tabs');
+  });
+
+  test('renders with Download__latest-version', () => {
+    render(<Tabs tabs={TestTabs} latestReleaseNumber={1} />);
+    const version = screen.getByTestId('Download__latest-version');
+    expect(version).toHaveClass('Download__latest-version');
+    expect(screen.getByText('Latest Version: 1.0.0-alpha.1')).toBeTruthy();
```

Comment:
minor improvement:

I think, on line 35 you can use `expect(version).toHaveTextContent()`